src/test/regress/expected/multi_subquery_toplevel_union.out

```diff
@@ -0,0 +1,777 @@
+--
```

Comment:
since we are steering away from 'toplevel_union' we can use multi_subquery_union